script.test.js

```diff
@@ -0,0 +1,298 @@
+import { test } from "uvu";
+import { equal, unreachable } from "uvu/assert";
+import { Octokit } from "@octoherd/cli";
+import { script } from "./script.js";
+import { repository } from "./tests/fixtures/respository-example.js";
+import nock from "nock";
+
+const getOctokitForTests = () => {
+  return new Octokit({
+    retry: { enabled: false },
+    throttle: { enabled: false },
+  });
```

Comment:
I tried to mute logs when running tests by adding `log: { ... }` here but was not working. Is there any easy way I can silence Octokit logs in this test?